web-apis/screen-capture/index.js

```diff
@@ -0,0 +1,59 @@
+const isSupported = () => {
+  return navigator.clipboard &&
+    navigator.clipboard.writeText &&
```

Comment:
@koustov change this with the screen capture's logic.

Reply:
@atapas updated.